Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/detail/AgendaDetailFragment.kt

```diff
@@ -32,16 +34,24 @@ import kotlinx.android.synthetic.main.agenda_detail_fragment.*
 
 
 class AgendaDetailFragment : Fragment() {
+    private val requireContext: Context
+        get() = context!!
```

Comment:
isn't there a 'requireContext' on the support fragment already?

Reply:
It's definitely there for Activity and I'm almost certain it's in Fragment as well now.
Yes, they added it because it was annoying to use nullable context in Kotlin when you knew for sure it was non-null.  "requireContext" is basically just an alias for "context!!".